# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../jadetex/compile-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur jadetex-3.13-orig/Makefile jadetex-3.13/Makefile
--- jadetex-3.13-orig/Makefile	2002-01-07 13:02:14.000000000 +0000
+++ jadetex-3.13/Makefile	2006-01-27 13:13:03.079847000 +0000
@@ -4,7 +4,7 @@
 	tex jadetex.ins
 
 jadetex.fmt: basic
-	tex -ini -progname=jadetex "&latex" jadetex.ini
+	latex -ini -progname=jadetex "&latex" jadetex.ini
 
 pdfjadetex.fmt: basic
 	pdftex -ini -progname=pdfjadetex "&pdflatex" pdfjadetex.ini
